lib/passport/serializers.coffee

```diff
@@ -18,8 +18,9 @@ module.exports.serialize = (user, done) ->
       request
         .get("#{opts.ARTSY_URL}/api/v1/me/authentications")
         .set('X-Access-Token': user.get 'accessToken').end(cb)
-  ], (err, [{ body: userData },{ body: authsData }]) ->
+  ], (err, results) ->
     return done err if err
+    [{ body: userData },{ body: authsData }] = results
```

Comment:
This was the culprit of today's issue. Destructuring the results before handling the error caused an issue if `err` existed unsurprisingly.